content/en/docs/concepts/security/index.md

```diff
@@ -632,331 +628,260 @@ authorization result, `ALLOW` or `DENY`.
-  name: default
+ name: httpbin
+ namespace: foo
 spec:
-  mode: 'ON_WITH_INCLUSION'
-  inclusion:
-    namespaces: ["default"]
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.auth.claims[iss]
+     values: ["https://accounts.google.com"]
 {{< /text >}}
 
 ### Authorization policy
 
-To configure an Istio authorization policy, you specify a `ServiceRole` and
-`ServiceRoleBinding`. Like other Istio configuration objects, they are
-defined as
-Kubernetes `CustomResourceDefinition` [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) objects.
+To configure an Istio authorization policy, you specify the [Authorization Policy](https://github.com/istio/api/blob/bfa91e88abf1af16d248917ebd3ab62036f7ca12/security/v1beta1/authorization.proto)
+configuration object. Like other Istio configuration objects, it is defined as Kubernetes `CustomResourceDefinition` [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+object.
 
-- **`ServiceRole`** defines a group of permissions to access services.
-- **`ServiceRoleBinding`** grants a `ServiceRole` to particular subjects, such
-  as a user, a group, or a service.
+An `AuthorizationPolicy` includes a `selector` and a list of `rules`, the `selector` specifies
+**where** to apply the policy and the `rules` specify **who** is allowed to do **what** under which **conditions**.
+Specifically:
 
-The combination of `ServiceRole` and `ServiceRoleBinding` specifies: **who** is
-allowed to do **what** under **which conditions**. Specifically:
+- **where** refers to the `selector` section in the `AuthorizationPolicy`.
+- **who** refers to the `from` section in the `rule` of the `AuthorizationPolicy`.
+- **what** refers to the `to` section in the `rule` of the `AuthorizationPolicy`.
+- **conditions** refers to the `when` section in the `rule` of the `AuthorizationPolicy`.
 
-- **who** refers to the `subjects` section in `ServiceRoleBinding`.
-- **what** refers to the `permissions` section in `ServiceRole`.
-- **which conditions** refers to the `conditions` section you can specify with
-  the [Istio attributes](/docs/reference/config/policy-and-telemetry/attribute-vocabulary/)
-  in either `ServiceRole` or `ServiceRoleBinding`.
+Workload selector decides where to apply the authorization policy. If not set,
+the authorization policy will be applied to all workloads in the same namespace
+as the authorization policy.
 
-#### `ServiceRole`
-
-A `ServiceRole` specification includes a list of `rules`, AKA permissions.
 Each rule has the following standard fields:
 
-- **`services`**: A list of service names. You can set the value to `*` to
-  include all services in the specified namespace.
+- **`from`**: A list of sources for specifying the identities.
```

Comment:
**`from`**: A list of `sources`.

Source does not only include identities. It can also be IPs, namespaces, etc.